spec/factories/blood_pressures.rb

```diff
@@ -32,6 +32,13 @@
       diastolic { rand(60..89) }
     end
   end
+
+  # We are making this as a new default BP factory to move to, because many of our tests
+  # will depend on the existance of an associated observation / encounter. We don't want to
+  # change the main factory as there would be too much test suite fallout.
+  factory :bp_with_encounter, parent: :blood_pressure do
+    with_encounter
+  end
```

Comment:
This doesn't feel very different from using the trait to me. What is the path you're envisioning to creating encounters by default?

Reply:
The goal is to have `with_encounter` be the default, but switch to that over time in multiple PRs.